docs/docs/tutorials/codealong/contract_tutorials/token_contract.md

```diff
@@ -65,35 +65,51 @@ There is one `initializer` function in this contract, and it will be selected an
 
 These are functions that have transparent logic, will execute in a publicly verifiable context and can update public storage.
 
-- `set_admin` enables the admin to be updated
-- `set_minter` enables accounts to be added / removed from the approved minter list
-- `mint_to_public` enables tokens to be minted to the public balance of an account
-- `mint_to_private` enables tokens to be minted to the private balance of an account (with some caveats we will dig into)
-- `transfer_to_public` enables tokens to be moved from a public balance to a private balance, not necessarily the same account (step 1 of a 2 step process)
-- `transfer_in_public` enables users to transfer tokens from one account's public balance to another account's public balance
-- `burn_public` enables users to burn tokens
+- [`set_admin`](#set_admin) enables the admin to be updated
+- [`set_minter](#set_minter)` enables accounts to be added / removed from the approved minter list
+- [`mint_to_public`](#mint_to_public) enables tokens to be minted to the public balance of an account
+- [`transfer_in_public`](#transfer_in_public) enables users to transfer tokens from one account's public balance to another account's public balance
+- [`burn_public`](#burn_public) enables users to burn tokens
+- [`finalize_mint_to_private`](#finalize_mint_to_private) finalizes a `prepare_private_balance_increase` call
+- [`finalize_transfer_to_private`](#finalize_transfer_to_private) finalizes a `prepare_private_balance_increase` call
 
 ### Private functions
 
 These are functions that have private logic and will be executed on user devices to maintain privacy. The only data that is submitted to the network is a proof of correct execution, new data commitments and nullifiers, so users will not reveal which contract they are interacting with or which function they are executing. The only information that will be revealed publicly is that someone executed a private transaction on Aztec.
 
-- `transfer` enables an account to send tokens from their private balance to another account's private balance
-- `transfer_in_private` enables an account to send tokens from another account's private balance to another account's private balance
-- `cancel_authwit` enables an account to cancel an authorization to spend tokens
-- `burn` enables tokens to be burned privately
+- [`transfer`](#transfer) enables an account to send tokens from their private balance to another account's private balance
+- [`transfer_in_private`](#transfer_in_private) enables an account to send tokens from another account's private balance to another account's private balance
+- [`transfer_to_private`](#transfer_to_private) transfers a specified `amount` from an accounts public balance to a designated recipient. This flow starts in private, but will be completed in public.
+- [`transfer_to_public`](#transfer_to_public) transfers tokens from a private balance, to a (potentially different account's) public balance
```

Comment:
Checking the code, `transfer_to_public` to also for transferring from another account's private balance (using authwits)